Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -35,6 +35,10 @@ public static class AdvancedFluentLayoutExtensions
         public static FluentLayout Above(this UIView view, UIView previous, nfloat? margin = null) =>
 			view.Bottom().EqualTo().TopOf(previous).Minus(margin.GetValueOrDefault(DefaultMargin));
 
+				public static FluentLayout CenterXAtTop(this UIView view, UIView previous) => view.CenterX().EqualTo().TopOf(previous);
+
+				public static FluentLayout CenterYAtTop(this UIView view, UIView previous) => view.CenterY().EqualTo().TopOf(previous);
```

Comment:
Idk why when we push to git, it mess up the indentation, even though it looks good on xamarin studio. That happens to me as well.